frontend/composables/notifications/useNotificationsManagementDashboards.ts

```diff
@@ -52,10 +59,99 @@ export function useNotificationsManagementDashboards() {
     },
     { immediate: true },
   )
+  const clearSettings = (
```

Comment:
@LuccaBitfly well this is a `code smell` in `frontend` and imo should be handled more gracefully by the `backend` one day